cogs/autorole.py

```diff
@@ -549,7 +570,8 @@ async def onboard_member(self, ctx, *users: str):
 
 2. Join our Steam Group <https://steamcommunity.com/groups/ghostproxy> 
 
-3. Head to the `#voluspa` channel and type `$help lfg-add` and then add your Game Mode roles
+3. Head to the `#voluspa` channel and type `$game-roles` and then add your Game Mode roles
+  (Type `$help game-roles` for more info, and also check out `$other-games` also)
```

Comment:
Noice - this looks like docs tidy after you brought d2 gamemodes across to gaminator?

Reply:
Correct, yah :+1: